samples/animated_gradient.py

```diff
@@ -1,12 +1,12 @@
 # This script simply sends an animated gradient to the virtual camera.
 
-import time
 import numpy as np
 import pyvirtualcam
 
-width = 1280
-height = 720
-fps = 20
+width_ = 1280
```

Comment:
I'm not a big fan of the trailing underscore. What's the reasoning behind it?

Reply:
Shadows name from outer scope.

Reply:
`default_width` might be used

Reply:
Or UPPER_CASE because it is a global constant.